### ISSUE TITLE
A11y_AzureToolsStorageExplorer_BreakLeasePopup_ScreenReader:Nvda does not announces the displayed status in the "Activities" window after activating the "ok" button.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/AzureStorageExplorer#2615]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
**GitHubTags:** #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #Win32; #Benchmark; #A11ySev2;  #MAS4.1.3; #NVDA; #JAWS;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
OS :Windows10
NVDA Version: 2019.2.1

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Right click on blob and select "Acquire Lease" control and hit enter.
7. Again right click on the blob select enabled "Break Lease" control and hit enter.
8. Start the NVDA.
9. Fill the details in the edit field present in the pop up.
10. Navigate to "Ok" button and press enter.
11. Observed the message which is displayed in the "activities" window does not announces by the screen reader

**Actual:**
The displayed message "Successfully broke lease for blob 'eere.txt' " in the activities window does not announces by the Nvda scree reader.

**Same issue observed for the below scenario:**
1.Acquire Lease
2.Clone with New Name Blob.
3.Rename Blob Container

**Expected:**
The displayed message "Successfully broke lease for blob 'eere.txt' " in the activities window should announces by the Nvda scree reader.

**User Impact:**
Screen reader user does not get the information about the action status regarding the control.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS4.1.3:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true

**Comment by vinodbk197** [comment]
We have uploaded repro video of this bug in [OneDrive](https://microsoft-my.sharepoint.com/personal/v-subha_microsoft_com/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fv%2Dsubha%5Fmicrosoft%5Fcom%2FDocuments%2FMicrosoft%20Azure%20Tools%20Storage%20Explorer%20Attachments&originalPath=aHR0cHM6Ly9taWNyb3NvZnQtbXkuc2hhcmVwb2ludC5jb20vOmY6L3Avdi1zdWJoYS9FdnhIZGcxemtOOVBrbkYxdU94bk9pUUJlanB4S1lnOW4yMUR5ZGdRX29tMGV3P3J0aW1lPWJNTVh6dTJ1MTBn) link with name "2615_A11y_AzureToolsStorageExplorer_BreakLeasePopup_ScreenReader" please refer the Screenshot for reference.
Please let us know if there is any problem in accessing the link

**Comment by 12-shweta** [comment]
Verified the bug in the version 1.16.0-dev
The issue is Fixed and does not reproduce.
Hence closing this issue.
Please find the below link for Closing attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-shwarh_microsoft_com/EXoatkLi8MNGhoa9ZIQpElYBBFZXCbSP5Bx0mLU7lz-pfQ?e=qeZWbA

**Comment by MRayermannMSFT** [comment]
Chitrarupa said this was fixed in email from 10/19.

**Comment by fsteffi** [comment]
**GitHub Tags:**#A11yMAS;#BM-Azure Tools Storage Explorer-win32-Aug2020;
